File: trunk/java/SR-RBD/src/ddb/restore/RestoreNack.java
package ddb.restore;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.MessageType;

public class RestoreNack extends RestoreMessage {

	@Override
	protected void fromBinary(DataInputStream s) throws IOException {
		// empty
	}

	@Override
	protected MessageType getType() {
		return MessageType.RESTORE_NACK;
	}

	@Override
	protected void toBinary(DataOutputStream s) throws IOException {
		// empty
	}

}

File: trunk/C#/RBD/RBD/Restore/Msg/RestoreNack.cs
using RBD.Communication;

namespace RBD.Restore.Msg
{
    public class RestoreNack : RestoreMessage {

        override public void FromBinary(DataInputStream s)
        {
		    // empty
	    }

        override public MessageType GetMessageType() {
		    return MessageType.RESTORE_NACK;
	    }

        override public void ToBinary(DataOutputStream s)
        {
		    // empty
	    }

    }
}
